README.md

```diff
@@ -81,38 +146,59 @@ will be processed to:
 }
 ```
 
-## Pseudo class and elements
+Psuedo-classes are also dealt with correctly, because it's easy to statically determine.
 
 ```css
 .foo {
-	--foo-color: #ff0000;
+	--foo-color: red;
 	color: var(--foo-color);
 }
 
 .foo:hover {
-	--foo-color: #00ff00;
+	--foo-color: green;
 }
 ```
 
-will be processed to:
+The above will be transformed to this:
```

Comment:
Let's standardize on `Will be transformed to:` language for all of before/after snippets